cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1505,6 +1505,12 @@ public static final class Logging {
     public static final String LOG_APPENDER_EXT_DIR = "app.program.log.appender.extensions.dir";
     public static final String LOG_APPENDER_PROPERTY_PREFIX = "app.program.log.appender.system.properties.";
 
+    // Log publisher configs.
+    public static final String LOG_PUBLISHER_PROVIDER = "app.program.log.publisher.provider";
```

Comment:
Remove app.program prefix. "app.program" is for runs. We are making a global instance-level mechanism here. Do simply "log.publisher".